lib/msf/core/web_services.rb

```diff
@@ -1,3 +1,3 @@
 module Msf::WebServices
-
+  extend Msf::WebServices::ModuleSearch
```

Comment:
Context: This was code was accidentally lost in a previous code shuffle here: https://github.com/rapid7/metasploit-framework/commit/1617b3ec9b336f38c071cc35de03191bd5cccc11#diff-14dba8397537f8e93680797413000a4cebec4393efb7f047a46fb0e997dc9719